mux.go

```diff
@@ -95,7 +95,8 @@ func newMux(dst string, option *ClientOption, init, dead wire, wireFn wireFn) *m
 	var multiplex int
 	if option.PipelineMultiplex >= 0 {
 		multiplex = 1 << option.PipelineMultiplex
-	} else {
+	}
+	if multiplex == 0 {
```

Comment:
Is this good way? Because we just replace wrong value with default, but user never knows about problem.
I think better panic with explanation why